server/runtime/src/main/resources/schema/infinispan-server-13.0.xsd

```diff
@@ -1596,4 +1596,13 @@
    <xs:complexType name="security-realm-trust">
       <xs:attribute type="xs:string" name="name"/>
    </xs:complexType>
+
+   <xs:attribute name="security-realm" type="xs:string">
```

Comment:
I believe this is not supposed to be here, right?

Reply:
Yes it is. The `security-realm` is in the `server` namespace but it applies to the `transport` which is declared in the `infinispan-config` schema (look for `<xs:anyAttribute/>`)